pos_multi_session/static/src/js/pos_multi_session.js

```diff
@@ -90,9 +90,14 @@ odoo.define('pos_multi_session', function(require){
                     self.get_bus('sync_server').add_channel_callback(channel_name, callback, self);
                     self.sync_bus = self.get_bus('sync_server');
                     self.get_bus('sync_server').start();
+                    if (!self.config.autostart_longpolling) {
+                        self.bus.activated = false;
+                        self.bus.start();
```

Comment:
Why do you start main longpolling if it's switched off in settings and another sync server is used?

Reply:
@yelizariev it's not possible to turn off the main longpolling, We can only turn off its autostart in longpolling.js

Reply:
Don't turn it off. Just don't start it

Reply:
Just remove this block